### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by ralphys**
restore errors and confirmation messages.
adjust widgets size
update help section
- minor wording updates
- list only common groups related to linux lite and remove inexistent
ones.
update about section
- set version number
- update credits

### CODE REVIEW

**Comment by shaggytwodope** [comment]
Copyright header goes back. There is no debate over this. I've told Jerry before that is the only legal catch to porting such a tool. If this is shipped to users it violates GPL.

**Comment by shaggytwodope** [comment]
@linuxlite You need to know that in this case the GPL states we can never remove any previous license. Even in the form of a header in the file. The one explicit thing I was told when I contacted FSF. Otherwise one can do anything they see fit, just no modifying the info in the header removing the original authors name and license.

**Comment by ralphys** [comment]
@shaggytwodope, to your first comment, I really don't have much to say but I hope you mean it in a positive way. Otherwise you are publicly making of yourself a rude and intolerable person.  

Regarding your second comment, it has been me the person who made the changes. Unfortunately, I did not add the original author in the 'header' and that's no nice but it does not violate the intended GPLv2 terms since the modified version does include the information in the software 'About' section and also carries the same license on distribution GPL-2(2ab). I will make sure to add the original author on the header also because it is the nice thing to do, as well as to include the date of my modification which is missing. Just because you stated that the author name and license has been removed it does not make it true. The modified distribution carries the same GPL license, properly states the author in the application and it is in public domain; we are not imposing restrictions in the software which is governed by the GPL license. All the best.

**Comment by shaggytwodope** [comment]
@ralphys that is an about, that is for the end user. NOT the source code it's self. Your are violating the gpl and I'm not debating this further.  You can remove the FULL header, note the gpl version and how to obtain the full license. But not remove it completely. Keeping all of the copyright info intact.

**You removed my name from a copyright header**

Read here before you make any assumptions, https://www.gnu.org/licenses/gpl-howto.en.html

Now that you've read this, you'll notice that removing MY name isn't going to fly.

At no point are you permitted to remove existing copyright notices. This is not a debatable matter.
Bottom line you can modify code all you want, you may not remove copyright info.

@linuxlite fork or not, it remains my work and this person removed my copyright. This is not funny, not should it be handled as such. Email me Jerry.

**Comment by ralphys** [comment]
@shaggytwodope 
I do not take this matter lightly since protecting free software is also a civil thing. The name Johnathan Jenkins has been always mentioned; credits have been given as it should. The link you provided only makes one thing clear to me; you do not read your own statements:

**"How to use GNU licenses for your own software**"
"... If several people helped write the code, use all their names"

Irrespective, I do respect your work as much as I respect my own, which is done wit the purpose of empowering users and contributing with the free software community. With that said, I don't understand the cause of your rage but I'm willing to help if it is in my power to do so. 

copyright

**Comment by shaggytwodope** [comment]
@ralphys the passive aggressive wording you are using ends now. I'm going to issue a dmca request for the github repo. Among other steps to be taken. You removed a copyright notice, and refuse to put it back. An about box is NOT copyright. It's merely noting who worked on it. "Linux Lite Dev Team" link is both broken, nor does not include my name on the new page https://www.linuxliteos.com/development.html anywhere. I gave you more then enough changes to fix. I'm following both GPL, copyright laws. And github's advice on how to deal with this. 

Bottom line, You messed up here removing my name. You fail to understand the difference of credit page in an about box. And copyright info in the header. You do not have the legal right to remove this at any stage. 

I gave many chances for you to fix this. I will get started on DMCA pull down request. As you do not understand what a copyright notice is. I'll be nice and explain, when I place my text with my work in the file. It stays. It has nothing to do with an about box. As the about only serves to inform the user. Not to notify someone working/looking at the code. Which is why it is in a header.

Last and only time I tell you this nicely, you removed copyright info with my name and year. You broke both GPL and copyright laws. If you want to talk about respect, then respect my wishes and add it back. Else in the morning I'll begin the process filing a DMCA request. And possibly other actions.

I've done all of this with respect with github's rules. My copyright notice was removed, this remains partly my work. And shall be treated as such. If the page linked to served to properly credit my work I'd have no issue. But as it stands at this very moment, the page linked does NOT properly give credit. And you removing my copyright line is not ok with me. 

You confuse copyright with credit, a simple and honest mistake if it is just that. But I will not play games here anymore. If this is not fixed by the morning. I'll address it as failure to comply with a request.

**Comment by shaggytwodope** [comment]
@ralphys more education for the sake of learning. The part of the GPL you are most interested in is [here](https://www.gnu.org/licenses/gpl-faq.html#GPLOmitPreamble), and copyright laws are pretty simple, you may never remove any copyright added by a previous author without removing all of his work. So I believe you misunderstand the difference between the header content (including the preamble) and an about box.
It is even agreeable to use something along the lines of:
```
# This file is covered by the LICENSE file in the root of this project.
``` 
If the license file is packaged, which in the case of linuxlite.. it is not.

I mean no offense when speaking to you, but this is an issue and one I do not take lightly. You violated laws and my own personal work. This is not open for debate, as laws are laid out fairly clearly in this regard. There are many past examples of companies doing this to very negative ends. You should know I did my research before even forking this project, so as to keep linuxlite from any legal issues. And as such I'm very well aware of what laws are in regards to this singular issue. 

You want to continue with childish words such as "rage", I can share words along those lines as well. But it should suffice when I tell you only once. You've violated laws here, and the GPL license. 

Maybe you googled around some, but you are by no means a lawyer. Nor am I for that matter. But all of your excuses and examples exclude actual laws. There is no situation when a copyright mention may be removed unless it is permitted by the author. Not at your whim, and your own personal logic. And I'll remind you for the last time, the about box is not at the top of the file, nor does it even include year. This means it's not following ANY law in doing so. While the intent is acknowledged, it's not how things are legally done.

You can't remove copyright info, as much as you would like too do so in order to clean up the python by a few bytes. It is not for you to decide.. ever. If you want more help or explaining in this subject I'm more then happy to explain and help you. But I won't debate something that I have full right to ask, even if it's not asked nicely.

**Comment by ralphys** [comment]
@shaggytwodope 
My passive aggressive wording is gonna end when you end yours. You shouldn't start a conversation the way you did without expecting others to treat you the same way. Feel free to do as you deem necessary. You can take any actions you see fit; I cannot prevent you from doing so.

You name is stated on both, the about section and also in the header; it has never been removed from the header in the first place. On GH, your name is also mentioned and linked to your profile:
https://github.com/linuxlite/liteusermanager

At least this time around you have mentioned what makes you unhappy: "...you removed copyright info with my name and year". I looked at what got removed and indeed, your claim of copyright was removed - Copyright 2013 Johnathan Jenkins <email@address>

I cannot say whether it was your sole work to claim such copyright or f it was you who infringed on the copyright of others by not mentioning them in it. With that said, my intention was to clean the file by adding: Copyright 2013-2017 (names - yours included as part of the Linux Lite Dev Team), but you have made clear that this is what bothers you and therefore I will keep that one line entirely.

I appreciate your attempt on schooling me on GPL. GPL is not as simple as you stated and Copyright is more complicated than claiming it because of some part of the code you changed to create another version. As a matter of fact, for any legal purpose such as a lawsuit you would have to register your copyright with the Copyright Office and each version by separate when it comes to software. Until you register, there’s nothing you can do to stop the infringement. Check with your lawyer. 

It would have saved us both the trouble if you had mentioned you wanted that one line to stay because you didn't feel your work was recognized otherwise. I'll add it back as a single line so that you don't feel your intellectual property rights has been violated.

**Comment by shaggytwodope** [comment]
You don't seem to know what the DMCA is. Nor do you know what copyright means.

Some nice reading for you https://www.plagiarismtoday.com/stopping-internet-plagiarism/your-copyrights-online/3-copyright-myths/

It is a complicated matter, but not devoid of any logic in what I've said. The GPL and copyright issues are two separate issues tho. Like I said you can't remove the header for GPL, you may however point to the license in another file making mention of it. And my work like anyone elses work here is instantly copyright. And copyright laws support me on this matter. And never added copyright for the group. It says modified by. This is a big difference. Wording means everything in copyright.

I do not need to register to perform a DMCA, this is well known. And how most things on the internet as whole works. I do however need to if I were to take legal actions in court.

Bottom line, as I stated before my copyright remains intact, there is no debate to be had here. You doing this violates both GPL and copyright law. As for what it takes for me to take action, Thats another matter entirely. It doesn't change anything at this point tho.

All of your points with credit are valid, but you misunderstand the nature of the program. It is to be shipped to users via deb packaging. And in the process renders most of the points you've made entirely moot. In fact it's violating the GPL right now by not shipping the license with the program and your current removal of the header. I think this is the base of your entire point here. And you may have overlooked this. An about box is not the only thing that is observed with these issues. Take a look at the debian camp and you'll see why it's an issue.

**Comment by ralphys** [comment]
And you seem to know what others know and what they don't. 

You are either plying dumb or it is not an act. The commit is on GH. A download includes the LICENSE with it and it hasn't been packaged yet because we are intending to add a **COPYING** file to each package. Please, go away. Or at least wait for us to violate the GPL license to actually substantiate your claim.

**Comment by shaggytwodope** [comment]
I'm not "plying" anything here, @linuxlite Jerry has not fixed this issue in two years. There is alot of reason to assume this. Basing future actions of previous knowledge is not wrong at all. You are assuming I'm new to how linuxlite works. Which you would be very wrong.

You are violating the GPL now, and with these changes even more so. Don't act high and mighty when you know full well you are wrong. Please go away yourself kiddo.

**Comment by linuxlite** [comment]
@shaggytwodope you seem to be obsessed with following us and what we do. It's kinda creepy. I know you're holding onto a grudge. Let it it go man. Not healthy.
